recipe/meta.yaml

```diff
@@ -46,7 +47,7 @@ outputs:
         - termcolor-cpp
         - cli11
         - fmt
-        - libiconv                               # [unix]
+        - libiconv                               # [osx]
```

Comment:
Maybe we can also get rid of `libiconv` here? I think it's a dependency of `libarchive` and we don't link with it directly.

Reply:
right, I wasn't brave enough to remove it completely.  But my tests have shown that there is neither a shared linking happening, nor a statically one

Reply:
done